pkg/clustermanager/cluster.go

```diff
@@ -15,6 +15,7 @@ type Manager struct {
 	nodes            []Node
 	clusterName      string
 	cloudInitFile    string
+	CNI              string
```

Comment:
In GoLang upper case capped fields are public, while lower case capped fields are private. In this type, we use mostly private fields, so `CNI` should be `cni`.

Reply:
Is fixed now